dev/doc/release-process.md

```diff
@@ -0,0 +1,96 @@
+# Release process #
+
+## As soon as the previous version branched off master ##
+
+- [ ] Create a new issue to track the release process where you can copy-paste
+  the present checklist.
+- [ ] Change the version name to the next major version and the magic numbers
+  (see [`e2f4973` + `3072bd9`](https://github.com/coq/coq/compare/5639933...3072bd9)
+  and [`244f32e`](https://github.com/coq/coq/commit/244f32e)).
+- [ ] Put the corresponding alpha tag using `git tag -s`.
+  The `VX.X+alpha` tag marks the first commit to be in `master` and not in the
+  branch of the previous version.
+- [ ] Create the `X.X+beta1` milestone if it did not already exist.
+- [ ] Decide the release calendar with the team (freeze date, beta date, final
+  release date) and put this information in the milestone (using the
+  description and due date fields).
+
+## About one month before the beta ##
+
+- [ ] Create the `X.X.0` milestone and set its due date.
+- [ ] Send an announcement of the upcoming freeze on Coqdev and ask people to
+  remove from the beta milestone what they already know won't be ready on time
+  (possibly postponing to the `X.X.0` milestone for minor bug fixes,
+  infrastructure and documentation updates).
+- [ ] Determine which issues should / must be fixed before the beta, add them
+  to the beta milestone, possibly with a
+  ["priority: blocker"](https://github.com/coq/coq/labels/priority%3A%20blocker)
+  label. Make sure that all these issues are assigned (and that the assignee
+  provides an ETA).
+- [ ] Ping the development coordinator (**@mattam82**) to get him started on
+  the update to the Credits chapter of the reference manual.
+  See also [#7058](https://github.com/coq/coq/issues/7058).
+  The command to get the list of contributors for this version is
+  `git shortlog -s -n VX.X+alpha..master | cut -f2 | sort -k 2`
```

Comment:
IIRC @cpitclaudel was missed by git for the sphinx stuff, maybe a note that the command yields an approximation would be good?

Reply:
Maybe, but I'd rather make sure that people properly set the "author" of their commits in the future.